source/isaaclab/isaaclab/sim/schemas/schemas_cfg.py

```diff
@@ -445,6 +445,8 @@ class MeshCollisionPropertiesCfg:
 
     physx_func: callable = MISSING
 
+    mesh_approximation_token: UsdPhysics.Tokens | PhysxSchema.Tokens = MISSING
```

Comment:
Can we add docstrings to all the attributes in this class? It seems previous MR missed out on this during the review process.